eZ/Publish/Core/MVC/Symfony/Routing/Tests/UrlAliasGeneratorTest.php

```diff
@@ -309,7 +310,7 @@ public function testDoGenerateNoUrlAlias()
             ->method('generate')
             ->with(
                 UrlAliasGenerator::INTERNAL_LOCATION_ROUTE,
-                array('locationId' => $location->id)
+                array('contentId' => $location->contentId, 'locationId' => $location->id)
```

Comment:
Actually, the idea is that `locationId` is an optional parameter. Haven't completely worked that part out yet :-)